src/azul/chalice.py

```diff
@@ -154,9 +181,13 @@ def _log_request(self):
                 # Convert MultiDict to a plain dict that can be converted to
                 # JSON. Also flatten the singleton values.
                 query = {k: v[0] if len(v) == 1 else v for k, v in ((k, query.getlist(k)) for k in query.keys())}
-            log.info(f"Received {context['httpMethod']} request "
+            token = self.current_auth_token
+            auth_status, auth_stmt = (('unauthenticated', '') if token is None
+                                      else ('authenticated', f' with token {token!r}'))
+            log.info(f"Received {auth_status} {context['httpMethod']} request "
```

Comment:
Is there a particular reason you're not following the C guide here?

https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#logging 4th bullet

Reply:
I was extrapolating from the existing code but there's no good reason to be using f-strings here.